singlestoredb/config.py

```diff
@@ -310,16 +310,16 @@
 register_option(
     'external_function.url', 'string', check_str, 'http://localhost:8000/invoke',
     'Specifies the URL of the external function application.',
-    environ=['SINGLESTOREDB_EXT_FUNC_URL'],
+    environ=['SINGLESTOREDB_EXT_FUNC_URL' ],
 )
 
 register_option(
     'external_function.app_mode', 'string',
     functools.partial(
         check_str,
-        valid_values=['remote', 'collocated'],
+        valid_values=['managed', 'remote', 'collocated'],
     ),
-    'remote',
+    'managed',
```

Comment:
I would leave the default as `remote` and set it explicitly to that in the application code. This app needs to run on-prem as well and the typical way is done using `remote`.